architecture.md

```diff
@@ -602,24 +602,33 @@ respectively create a virtual machine or a container inside a virtual machine as
 with a Kubernetes pod:
 
 ```Go
-	containerType, err := ociSpec.ContainerType()
+	containerType, err := oci.GetContainerType(status.Annotations)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
-	switch containerType {
-	case vc.PodSandbox:
-		process, err = createPod(ociSpec, runtimeConfig, containerID, bundlePath, console, disableOutput)
+	ociSpec, err := oci.GetOCIConfig(status)
+	if err != nil {
+		return nil, err
+	}
+
+	var sandbox vc.VCSandbox
+
+	if containerType.IsSandbox() {
+		s, err := vci.StartSandbox(ctx, sandboxID)
```

Comment:
I feel that you updated with the wrong code here. Did you use `cli/start.go` instead of `cli/create.go`?

Reply:
Yes, u are right.Thx for the sugguestion.
I update the change.